modules/create-data-object/pkg/utils/parse/clioptions.go

```diff
@@ -63,8 +80,10 @@ func (c *CLIOptions) Init() error {
 		return err
 	}
 
-	if err := c.assertValidTypes(); err != nil {
-		return err
+	if !c.GetDeleteObject() {
```

Comment:
This shadows the assertion output type.

Reply:
The output type is not used during delete operation